pkg/controller/watermarkpodautoscaler/replica_calculator.go

```diff
@@ -232,17 +237,35 @@ func (c *ReplicaCalculator) getReadyPodsCount(namespace string, selector labels.
 			toleratedAsReadyPodCount++
 		}
 	}
+	log.Info("getReadyPodsCount", "full podList length", len(podList), "toleratedAsReadyPodCount", toleratedAsReadyPodCount, "incorrect target", incorrectTargetPods)
```

Comment:
similarly, perhaps `incorrect target` -> `incorrect target pod count`

Reply:
could it be worth logging the target name?

Reply:
We log the target name a little before, the info I am trying to display is how many pods share the same label selector essentially.

Reply:
will use `incorrectly targeted pods`